www/js/index.js

```diff
@@ -12,6 +12,8 @@ function escKey() {
   /* if (settings.$settings.classList.contains('animated')) {
     settings.closeSettings();
   } */
+  // show Empty Slide
+  global.controller.sendText('');
```

Comment:
This is being bound with bindGlobal, which will tigger this even if `esc` is pressed while the cursor is in a text field. That doesn't seem intuitive. @maneetpaul @saintsoldierx Do you have any thoughts on what criteria need to be met for the `esc` key to trigger an empty slide?

Reply:
I agree! Its bad UX to repurpose keystrokes like that. I think we should do something like `cmd + e` or `ctrl+e` for this.

Reply:
This was later decided to be cmd/ctrl+e, so I will click "resolve" for this conversation